dodola/core.py

```diff
@@ -296,7 +299,7 @@ def standardize_gcm(ds, leapday_removal=True):
         ds_noleap = xclim_remove_leapdays(ds_cleaned)
 
         # rechunk, otherwise chunks are different sizes
-        ds_out = ds_noleap.chunk(730, len(ds.lat), len(ds.lon))
+        ds_out = ds_noleap.chunk({"time": 730, "lat": len(ds.lat), "lon": len(ds.lon)})
```

Comment:
Do we want to hard-code the dim names here?

Reply:
@dgergel I'm likely going to merge this as is, but I'm happy to reopen and adjust this later.

It was hardcoding dim names or hard coding dim order.

My thinking was that it's better to have it fail loudly on explicit names than quietly run and rechunk things if the order of dimensions is actually different.